pages/_document.js

```diff
@@ -3,29 +3,6 @@ import Document, { Html, Head, Main, NextScript } from 'next/document'
 import { ServerStyleSheet } from 'styled-components/macro'
 import env from 'lib/environment'
 
-const ANALYTICS_CODE = `
```

Comment:
The account module has one place where we're tracking the type of account selected, we should remove that too.

Reply:
Solved in [`ddb920c`](https://github.com/aragon/bonding.aragon.org/pull/3/commits/ddb920c250993cd9e8ef5e92c76c919853f57d70)